check-redis/lib/check-redis.go

```diff
@@ -63,6 +64,14 @@ func connectRedis(m redisSetting) (*redis.Client, error) {
 	if err != nil {
 		return nil, fmt.Errorf("couldn't connect: %s", err)
 	}
+
+	if m.Password != "" {
+		r := c.Cmd("AUTH", m.Password)
+		if r.Err != nil {
+			return nil, errors.New("couldn't execute query")
```

Comment:
Do this *r.Err* contain the case of authentication failed by password incorrect? If so this error message is not kindly. I think it is better that an error includes an error from Redis, like `fmt.Errorf("couldn't authenticate: %v", r.Err)`.

Reply:
>  I think it is better that an error includes an error from Redis

Right! Fixed :)